src/Components/Nav/Nav.js

```diff
@@ -1,7 +1,241 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import { BiSearch, BiCart } from 'react-icons/bi';
+import { BsBookmark, BsBell } from 'react-icons/bs';
+import { CgProfile } from 'react-icons/cg';
+import { MdPhotoCamera } from 'react-icons/md';
+import { AiOutlineDown } from 'react-icons/ai';
+import NAV_DATA from './navdata';
+import Category from './components/Category';
 
 function Nav() {
-  return <div></div>;
+  const [categoryShow, setCategoryShow] = useState(false);
+
+  const handleCategoryOn = () => {
+    setCategoryShow({
+      categoryShow: true,
+    });
+  };
+  const handleCategoryOff = () => {
+    setCategoryShow({
+      categoryShow: false,
+    });
+  };
+  console.log(categoryShow);
+  return (
+    <div>
+      <HeadNav>
+        <NavBar>
+          <NavMain>
+            <div>
+              <Link to="/">
+                <MainLogo alt="오늘의 집" src="/images/logo.png" />
+              </Link>
+            </div>
+            <NavMainContent>
+              <Link
+                onMouseEnter={handleCategoryOn}
+                onMouseLeave={handleCategoryOff}
+                className="mainContent-menu"
+                to="/"
+              >
+                커뮤니티
+              </Link>
+            </NavMainContent>
+            <NavMainContent>
+              <Link className="mainContent-menu" to="/">
+                스토어
+              </Link>
+            </NavMainContent>
+            <NavMainContent>
+              <Link className="mainContent-menu" to="/">
+                인테리어 시공
+              </Link>
+            </NavMainContent>
+          </NavMain>
+          <NavMain>
+            <div>
+              <form className="form">
+                <input placeholder="스토어검색" type="text" required />
+                <button type="submit">
+                  <BiSearchs className="inputIcon"></BiSearchs>
+                </button>
+                <button type="submit">
+                  <MdPhotoCameras></MdPhotoCameras>
+                </button>
+              </form>
+            </div>
+            <div>
+              <Link to="/">
+                <BsBookmark className="Nav-Aside"></BsBookmark>
+              </Link>
+            </div>
```

Comment:
```suggestion
            <NavAside button={<BsBookmark />} />
```
- 이런 식으로 구성될 수 있도록 컴포넌트 분리하고 반복문으로 처리해보면 어떨까요?
- 셀프 클로징 태그 적용할 수 있는 곳에는 적극 사용해주세요.

Reply:
종택님의 도움으로 코드 깔끔하게 정리하였습니다!